openapi/api.yml

```diff
@@ -932,3 +932,75 @@ paths:
                 code: 401
                 status: 'error'
                 message: 'Could not save Flag for application'
+
+  '/qa_complete/{application_id}/{user_id}':
+    post:
+      tags:
+        - qa_complete
+      summary: create a qa_complete record for an application
+      description: post a qa_complete record for an application
+      operationId: api.routes.post_qa_complete_for_application
+      parameters:
+      - name: application_id
+        in: path
+        required: true
+        schema:
+          type: string
+          format: path
+      - name: user_id
+        in: path
+        required: true
+        schema:
+          type: string
+          format: path
+      responses:
+        201:
+          description: SUCCESS - Application updated to QA_COMPLETE
+          content:
+            application/json:
+              schema:
+                type: object
+        401:
+          description: ERROR - Could not mark application as QA_COMPLETE
+          content:
+            application/json:
+              schema:
+                $ref: 'components.yml#/components/schemas/Error'
+              example:
+                code: 401
+                status: 'error'
+                message: 'Could not mark application as QA_COMPLETE'
+
+  '/qa_complete/{application_id}':
```

Comment:
this will only get qa info for the old qa flag, once we remove that we'll need to use this new endpoint that searches if there is a qa_complete record for an application 👍🏾